test/kubemark/start-kubemark.sh

```diff
@@ -221,7 +221,7 @@ function create-and-upload-hollow-node-image {
 function create-and-upload-hollow-node-image-bazel {
   RETRIES=3
   for attempt in $(seq 1 ${RETRIES}); do
-    if ! bazel run //cluster/images/kubemark:push --define PROJECT="${PROJECT}"; then
+    if ! bazel run //cluster/images/kubemark:push --define REGISTRY="${FULL_REGISTRY}"; then
```

Comment:
This works with the current implementation of docker_build, since it's just concatenating the strings under the covers, but it would be good to be able to be able to assign `$PROJECT/` as a base repository for `kubemark` in the BUILD target, in case that ever changes in the future.

Reply:
?  PROJECT is a faux construct, AFAICT.  gcr.io/google_containers is the registry.  If you want something else, specify the whole thing.

Reply:
In the Docker sense, 'registry' == `hostname[:port]` and 'repository'  == `path` of the URLesque image name: https://docs.docker.com/registry/spec/api/#overview
I'm worried that the a future implementation of `docker_push` might make use of that assumption.

Suggestions added in the relevant places...